test/commands/generate/fromTemplate.test.ts

```diff
@@ -25,20 +25,6 @@ describe('template', () => {
       done();
     });
 
-  test
```

Comment:
@Souvikns and I have been tinkering with this test a lot lately. the thing is that this test fails randomly. in local and in [CI/CD pipeline](https://github.com/asyncapi/cli/actions/runs/3693882406/jobs/6254400684). We thought it would be better to get rid of it.

Reply:
instead of removing, in separate PR probably, we need just a fix for test. I remember having lots of random fails on `diff` command tests and we had to rework these to have each test in separate `describe`.

in short https://github.com/asyncapi/cli/blob/master/test/commands/generate/fromTemplate.test.ts#L17-L56 -> all 3 tests should be separated in individual `describe` as they probably overlap each other because of some async shit 😄